cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -222,7 +222,10 @@ func main() {
 		klog.V(2).Infof("Supports migration from in-tree plugin: %s", supportsMigrationFromInTreePluginName)
 
 		// Create a new connection with the metrics manager with migrated label
-		metricsManager = metrics.NewCSIMetricsManagerWithOptions(provisionerName, metrics.WithMigration())
+		metricsManager = metrics.NewCSIMetricsManagerWithOptions(provisionerName,
+			// Will be provided via default gatherer.
+			metrics.WithProcessStartTime(false),
```

Comment:
Should we remove the processstarttime metric from csi-lib-utils now that it's included by default?

Reply:
Some sidecars only use the csi-lib-utils code and nothing else, in which case processtarttime still needs to be added there.

Reply:
I thought the plan was to also add the base metrics to all sidecars?

Reply:
Yes, but that's just a plan. I don't know who's working on it. We can only remove it once all sidecars are updated.